connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/connect/client/util/RemoteSparkSession.scala

```diff
@@ -168,6 +159,56 @@ trait RemoteSparkSession extends ConnectFunSuite with BeforeAndAfterAll {
   import SparkConnectServerUtils._
   var spark: SparkSession = _
   protected lazy val serverPort: Int = port
+  protected lazy val transferredClientJarIfNeed: Boolean = {
+    if (sys.env.contains("USE_MAVEN")) {
+      if (RemoteSparkSession.transferredClientJar.get()) {
```

Comment:
It seems that the same `Artifact` cannot be transmitted repeatedly, so add a status to ensure this